rdkPlugins/Minidump/source/README.md

```diff
@@ -0,0 +1,41 @@
+# Dobby RDK Minidump Plugin
+
+## Quick Start
+The main task of Minidump Plugin is to allow the collection of minidump core files from container bounds.
```

Comment:
Please add a note that the app must be:

* Compiled with breakpad support
* Must be using the latest version of breakpad_wrapper with the `BREAKPAD_FD` support